drivers/hud_apr/app/models/hud_apr/generators/shared/fy2024/question_six.rb

```diff
@@ -0,0 +1,908 @@
+      answer = @report.answer(question: table_name, cell: 'B5')
+
+      dkr_members = dq_universe_members.where(race_col.in(['8', '9']))
+      answer.add_members(dkr_members)
+      answer.update(summary: dkr_members.count)
+
+      # Race missing
+      answer = @report.answer(question: table_name, cell: 'C5')
+      m_members = dq_universe_members.where(race_col.eq('99'))
+      answer.add_members(m_members)
+      answer.update(summary: m_members.count)
+
+      # Total
+      answer = @report.answer(question: table_name, cell: 'E5')
+      total_members = dkr_members.or(m_members)
+      answer.add_members(total_members)
+      answer.update(summary: total_members.count)
+
+      # Percentage
+      answer = @report.answer(question: table_name, cell: 'F5')
+      answer.update(summary: percentage(total_members.count / dq_universe_members.count.to_f))
+
+      total_members
+    end
+
+    private def simple_quality(table_name:, row:, attr:)
+      row_label = row.to_s
+      # DK/R
+      answer = @report.answer(question: table_name, cell: 'B' + row_label)
+      dkr_members = dq_universe_members.where(a_t[attr].in([8, 9]))
+      answer.add_members(dkr_members)
+      answer.update(summary: dkr_members.count)
+
+      # Missing
+      answer = @report.answer(question: table_name, cell: 'C' + row_label)
+      m_members = dq_universe_members.where(
+        a_t[attr].eq(nil).
+          or(a_t[attr].eq(99)),
+      )
+      answer.add_members(m_members)
+      answer.update(summary: m_members.count)
+
+      # Total
+      answer = @report.answer(question: table_name, cell: 'E' + row_label)
+      total_members = dkr_members.or(m_members)
+      answer.add_members(total_members)
+      answer.update(summary: total_members.count)
+
+      # Percentage
+      answer = @report.answer(question: table_name, cell: 'F' + row_label)
+      answer.update(summary: percentage(total_members.count / dq_universe_members.count.to_f))
+
+      total_members
+    end
+
+    ###
+    # Q6b
+    ###
+
+    def q6b_universal_data_elements
+      table_name = 'Q6b'
+      metadata = {
+        header_row: [
+          'Data Element',
+          NO_CLIENT_ANSWER_DESC,
+          INFORMATION_MISSING_DESC,
+          'Data Issues',
+          'Total',
+          '% of Issue Rate',
+        ],
+        row_labels: [
+          'Veteran Status (3.07)',
+          'Project Start Date (3.10)',
+          'Relationship to Head of Household (3.15)',
+          'Enrollment CoC (3.16)',
+          'Disabling Condition (3.08)',
+        ],
+        first_column: 'B',
+        last_column: 'F',
+        first_row: 2,
+        last_row: 6,
+      }
+      @report.answer(question: table_name).update(metadata: metadata)
+
+      sheet = question_sheet(question: table_name)
+      q6b_veteran_row(sheet)
+      q6b_project_row(sheet)
+      q6b_hoh_relationship_row(sheet)
+      q6b_client_location_row(sheet)
+      q6b_disabling_condition_row(sheet)
```

Comment:
do these need to return the sheet?
```suggestion
      q6b_veteran_row(sheet)
      q6b_project_row(sheet)
      q6b_hoh_relationship_row(sheet)
      q6b_client_location_row(sheet)
      q6b_disabling_condition_row(sheet)
```